### DISCUSSION TITLE
Mediamtx.yml raspberry pi after changes to paths (installation guide outdated?)

### DISCUSSION THREAD

**Discussion by crevatsch** [bluenviron/mediamtx#2856] [Questions]
### Question

Hi everyone! I've been using mediamtx 1.0.0 on a raspberry pi zero 24/7 for months now without problems so i'm very happy with the project! So happy in fact that i want to add a second camera. While installing the newest version (1.4.1) i'm having trouble getting mediamtx to work. It used to be that you had to add ''cam'' and ''rpiCamera'' to paths. Since a few weeks the paths section has been moved to the bottom of the mediamtx.yml. The install guide still contains the following:

3. edit mediamtx.yml and replace everything inside section paths with the following content:
paths:
  cam:
    source: rpiCamera

Is this still accurate after the paths section has been moved?

**Comment by aler9** [comment]
Hello, the configuration you posted is still accurate, since backward compatibility has always been guaranteed. If you have problems, open another issue and provide a server log with logLevel set to debug as explicitly mentioned in the instructions.

[a comment by an automated account is not shown]
